### PR TITLE
LPS-167708 Fix RaylifeSite#CanRaylifeBeCreated

### PR DESCRIPTION
**Pull request by itstali**

### CODE REVIEW

**Comment by itstali** [comment]
ci:test:site-initializer-raylife-d2c:nocompile

**Comment by itstali** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/display_layout/edit_cp_definition_cp_display_layout.jsp [hunk 1]
```diff
@@ -74,7 +74,7 @@ String searchContainerId = "CPDefinitionsSearchContainer";
 		<aui:model-context bean="<%= cpDisplayLayout %>" model="<%= CPDisplayLayout.class %>" />
 
 		<div class="sheet">
-				<div class="panel-group panel-group-flush">
+			<div class="panel-group panel-group-flush">
```
Comment (barrosdennys) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/display_layout/edit_cp_definition_cp_display_layout.jsp:77]: Did you really change this or was it by mistake?

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 2]
```diff
@@ -3010,8 +3010,7 @@ private void _validateObjectStateTransition(
 		ListTypeEntry originalListTypeEntry =
 			_listTypeEntryLocalService.getListTypeEntry(
 				listTypeDefinitionId,
-				_getValue(
-					String.valueOf(values.get(entry.getKey()))));
+				_getValue(String.valueOf(values.get(entry.getKey()))));
```
Comment (barrosdennys) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:3013]: Did you really change this or was it by mistake?

File: modules/apps/site-initializer/site-initializer-raylife-d2c/site-initializer-raylife-d2c-test/src/testFunctional/tests/RaylifeSite.testcase [hunk 3]
```diff
@@ -3,7 +3,7 @@ definition {
 
 	property osgi.modules.includes = "site-initializer-raylife-d2c";
 	property portal.release = "false";
-	property portal.upstream = "true";
+	property portal.upstream = "talitha";
```
Comment (barrosdennys) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/site-initializer-raylife-d2c-test/src/testFunctional/tests/RaylifeSite.testcase:6]: Hi @itstali 
This value of portal.upstream can't be merged. Would you please change back to "true"?

File: modules/apps/site-initializer/site-initializer-raylife-d2c/test.properties [hunk 4]
```diff
@@ -23,7 +23,7 @@
     #
 
     test.batch.run.property.query[functional-tomcat*-mysql*-jdk8][relevant]=\
-        (portal.acceptance == true) AND \
+        (portal.acceptance == talitha) AND \
```
Comment (barrosdennys) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/test.properties:26]: Hi @itstali
This value of portal.upstream can't be merged. Would you please change back to "true"?

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java (hunk 2 above)
Comment (itstali) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:3013]: Not really, probably a change from SF.

File: modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/display_layout/edit_cp_definition_cp_display_layout.jsp (hunk 1 above)
Comment (itstali) [reply modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/display_layout/edit_cp_definition_cp_display_layout.jsp:77]: Not really, probably a change from SF.

**Comment by itstali** [comment]
ci:test:sf

**Comment by itstali** [comment]
ci:test:sf

**Comment by barrosdennys** [comment]
ci:forward

**Pull request closed without merging**